W-15505961-azure-key-vault-attributes-kt (#2558)
mulesoft/docs-connectors: azure-key-vault-properties-provider/2.0/modules/ROOT/pages/azure-key-vault-properties-provider-reference.adoc

> For authenticating to Azure Key Vault using environment variables, you must set the following environment variables `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, and `AZURE_TENANT_ID`. Refer to xref:mule-runtime::setting-environment-variables.adoc[] for details.

Reviewer: ```suggestion
For authenticating to Azure Key Vault using environment variables, set the following environment variables `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, and `AZURE_TENANT_ID`. Refer to xref:mule-runtime::setting-environment-variables.adoc[] for details.
```

> In addition to setting the three environment variables, in the XML connector configuration you must set the following three attributes with an empty string: `tenantId=""`, `clientId=""`, and `clientSecret=""`.

Reviewer: ```suggestion
After setting the environment variables, in the XML connector configuration, set the following attributes with an empty string: `tenantId=""`, `clientId=""`, and `clientSecret=""`.
```
